src/routes/Router.ts

```diff
@@ -60,10 +63,11 @@ router.post(`${routesConfig.config.datasource.list.path}`, ResponseHandler.setAp
 
 /** Exhaust API(s) */
 router.get(`${routesConfig.exhaust.path}`, ResponseHandler.setApiId(routesConfig.exhaust.api_id), validationService.validateRequestParams, exhaustService.getData);
+router.get(`${routesConfig.prometheus.path}`, metricsScrapeHandler);
 
 /** Query Wrapper API(s) */
-router.post(routesConfig.query_wrapper.sql_wrapper.path, wrapperService.forwardSql)
-router.post(routesConfig.query_wrapper.native_post.path, wrapperService.forwardNative)
-router.get(routesConfig.query_wrapper.native_get.path, wrapperService.forwardNativeGet)
-router.delete(routesConfig.query_wrapper.native_delete.path, wrapperService.forwardNativeDel)
-router.get(`/status`, wrapperService.nativeStatus)
+router.post(routesConfig.query_wrapper.sql_wrapper.path, ResponseHandler.setApiId(routesConfig.query_wrapper.sql_wrapper.api_id), onRequest({ entity: promEntities.data_out }), wrapperService.forwardSql)
+router.post(routesConfig.query_wrapper.native_post.path, ResponseHandler.setApiId(routesConfig.query_wrapper.native_post.api_id), onRequest({ entity: promEntities.data_out }), wrapperService.forwardNative)
+router.get(routesConfig.query_wrapper.native_get.path, ResponseHandler.setApiId(routesConfig.query_wrapper.native_get.api_id), onRequest({ entity: promEntities.data_out }), wrapperService.forwardNativeGet)
+router.delete(routesConfig.query_wrapper.native_delete.path, ResponseHandler.setApiId(routesConfig.query_wrapper.native_delete.api_id), onRequest({ entity: promEntities.data_out }), wrapperService.forwardNativeDel)
+router.get(routesConfig.query_wrapper.druid_status.path, ResponseHandler.setApiId(routesConfig.query_wrapper.druid_status.api_id), onRequest({ entity: promEntities.data_out }), wrapperService.nativeStatus)
```

Comment:
Remove the metrics scrapper middleware for the status route, since it does not fall under both data in and data out scenario.